.github/workflows/workflow.yaml

```diff
@@ -65,7 +65,8 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        gatekeeper: [ "release-3.13", "release-3.14" ]
+        gatekeeper: [ "3.14.2", "3.15.1" ]
```

Comment:
why do we prefer to use specific patch versions instead of the latest?

Reply:
AFAIK, we have always tested with past two release versions. I am fine with testing it with latest, unless others have any objections.

Reply:
using the branch instead of the tag ensures we are always testing against the latest patch version without having to push a PR to update it.

Reply:
AFAIK helm can only install specifi version, I am not sure if it can take a branch and install the latest patch on that branch. If we do not use helm, then we would need to use `sed` or something similar to modify the file and set the flag manually.

Reply:
+1 to @JaydipGabani i think this is a better approach now since we previously installed from yaml directly

Reply:
ah I see. I'm ok with merging this as is. but it would be better to not having to bump the tag all the time. @JaydipGabani you can create an issue to do it as a follow up.

Reply:
Sure!

Reply:
I don't think we need to bump the patch version. We'll need to bump the minor version, which is same as today